src/views/standalone/security/ThreatShieldView.vue

```diff
@@ -10,6 +10,12 @@ import { useTabs } from '@/composables/useTabs'
 import AllowlistTab from '@/components/standalone/security/threat_shield/AllowlistTab.vue'
 import BlocklistTab from '@/components/standalone/security/threat_shield/BlocklistTab.vue'
 import SettingsTab from '@/components/standalone/security/threat_shield/SettingsTab.vue'
+import LocalBlocklistTab from '@/components/standalone/security/threat_shield/LocalBlocklistTab.vue'
+
+export type BlockOrAllowAddress = {
+  address: string
+  description: string
+}
```

Comment:
Maybe `BanIpLocalAddress` is better?

It's your call on this one

Reply:
Done